Document/0x06c-Reverse-Engineering-and-Tampering.md

```diff
@@ -50,6 +50,56 @@ Building a reverse engineering environment for free is possible. However, there
 
 Because Objective-C and Swift are fundamentally different, the programming language in which the app is written affects the possibilities for reverse engineering it. For example, Objective-C allows method invocations to be changed at runtime. This makes hooking into other app functions (a technique heavily used by [Cycript](http://www.cycript.org/ "Cycript") and other reverse engineering tools) easy. This "method swizzling" is not implemented the same way in Swift, and the difference makes the technique harder to execute with Swift than with Objective-C.
 
+In iOS, all the application code, for both Swift and Objective-C, is compiled to native, unlike Android where code is compiled into Dalvik Bytecode and native code (if code written in C/C++). Thus, to analyze iOS application a disassembler is needed. 
```

Comment:
A small suggestion as we say in the introduction that `native code` is c/c++. instead let's use machine code here:
```suggestion
In iOS, all the application code, for both Swift and Objective-C, is compiled to machine code (e.g. ARM). Thus, to analyze iOS application a disassembler is needed. 
```